### PR TITLE
Rich text support: make the formatting toolbar appear on demand #5775

### LINKED ISSUE #5775: Rich text support: make the formatting toolbar appear on demand
The toolbar covering parts of the form look a bit untidy.
image

Should we show a small icon clicking of which inserts the toolbar at the top of the text box (without covering the other things)?

### PR DESCRIPTION
**Pull request by belyabl9**
Fixes #5775

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 1]
```diff
@@ -122,6 +122,7 @@
                                 </label>
                             </h5>
                             <div class="col-sm-10">
+                                <div id="richToolbarContainer"></div>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Use `this-kind-of-case` for div IDs (although there are myriad of violations across the codebase; we're currently working to fix that), so this one should be `richtext-toolbar-container`.

**Comment by wkurniawan07** [comment]
Minor change requested, otherwise it looks good :+1:

**Comment by belyabl9** [comment]
Ready to review

**Comment by wkurniawan07** [comment]
This one's good to go as well :+1:

**Comment by wkurniawan07** [comment]
@damithc can this one go to the current release as well?

**Comment by damithc** [comment]
Yes, it can go in this release.

**Pull request merged**